README.md

```diff
@@ -64,16 +64,19 @@ In current `tsconfig.json` add:
 
 ## Why do we have that dependencies?
```

Comment:
If you rename readme files to `README.ru.md`, you should not forget to create `README.md` so that it can be shown automatically for the directory.